plugins/ssl/ssl-certificate-expiry

```diff
@@ -97,7 +98,13 @@ parse_valid_days_from_certificate() {
     input_data=$(cat)
 
     if echo "$input_data" | grep -q -- "-----BEGIN CERTIFICATE-----"; then
-        valid_until_string=$(echo "$input_data" | openssl x509 -noout -enddate \
+        cert_data=$(echo "$input_data" | openssl x509 -noout -subject_hash -enddate)
+
+        # Skip certificate if its hash is in env.skip_cert_hashes
+        hash="$(echo "${cert_data}" | head -n 1)"
+        echo "${skip_cert_hashes:-}" | grep -q "${hash}" && return
```

Comment:
How about `grep -iqwF` instead?

At least `-i` feels reasonable to me - the rest is just obsession with details. Pick you path :)